jitsi_uploader.sh

```diff
@@ -122,7 +119,15 @@ function process_upload_dir {
 echo $(date) "START Uploader tool received path \"$UPLOAD_DIR\""
 echo $(date) $(ls -l $UPLOAD_DIR 2>&1)
 
-process_upload_dir $UPLOAD_DIR
+# main
+if [[ $UPLOAD_TYPE == "dropbox" ]] && [[ ! -z $TOKEN ]]; then
+  process_upload_dir $UPLOAD_DIR
+elif [[ $UPLOAD_TYPE == "custom" ]]; then
+  custom_upload
```

Comment:
The process_upload_dir should be used in both cases, with either dropbox and a token, OR custom.  No need to call custom_upload here as you added a handler for it in the switch statement that says UPLOAD_FUNC.